### ISSUE TITLE
Uncaught ReferenceError: swal is not defined

### ISSUE THREAD

**Issue by norbertbilbeny** [t4t5/sweetalert#662]
In production I can work with Sweet Alert but when I put the site up, I get that error.

I ran `npm install sweetalert` and added these two lines to the <head> of index.html:

> <script `src="node_modules/sweetalert/dist/sweetalert.min.js"></script>
>  <link` rel="stylesheet" href="node_modules/sweetalert/dist/sweetalert.css">

Is that where they should be? Should they be elsewhere / nowhere?

When the page loads, Chrome's console shows these three errors: 

> GET https://wfolly.firebaseapp.com/node_modules/sweetalert/dist/sweetalert.css 
wfolly.firebaseapp.com/:24 
GET https://wfolly.firebaseapp.com/node_modules/sweetalert/dist/sweetalert.min.js 
wfolly.firebaseapp.com/:25 
GET https://wfolly.firebaseapp.com/node_modules/sweetalert/dist/sweetalert.css 

I'm a novice. A basic mistake is likely.

Ideas welcome.

**Comment by xlcrr** [comment]
Did you find out how to reference the files?

**Comment by norbertbilbeny** [comment]
I ended up using these two lines in index.html 

`    <script src="https://cdnjs.cloudflare.com/ajax/libs/sweetalert/1.1.3/sweetalert.min.js"></script>
    <link rel="stylesheet" href="node_modules/sweetalert/dist/sweetalert.css">`

and when pushing the build to (in my case) Firebase, I had to keep that single css file in the node_modules folder/etc directory. It is the only file in that folder that I have to keep even after the build to make it work. I don't know why it doesn't include it in the normal build process but at least it works this way.

Are you having problems?
